Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -226,84 +168,9 @@ public void Roslyn_Settings_ErrorLogAlreadySet()
             logger.AssertTargetExecuted(TargetConstants.SetRoslynAnalysisPropertiesTarget);
             BuildAssertions.AssertTargetSucceeded(result, TargetConstants.OverrideRoslynAnalysisTarget);
 
-            // Check the intermediate working properties have the expected values
-            BuildAssertions.AssertExpectedPropertyValue(result.ProjectStateAfterBuild, "SonarQubeRoslynRulesetExists", "True");
-            BuildAssertions.AssertExpectedPropertyValue(result.ProjectStateAfterBuild, "SonarQubeRoslynAssemblyExists", "True");
-            BuildAssertions.AssertExpectedPropertyValue(result.ProjectStateAfterBuild, "SonarLintFound", "true");
-
             BuildAssertions.AssertExpectedPropertyValue(result.ProjectStateAfterBuild, TargetProperties.ErrorLog, "already.set.txt");
         }
-
-        [TestMethod]
-        [Description("Checks the analyzer assembly is added if the analyzer assembly exists")]
-        public void Roslyn_Settings_Analyzer_AssemblyExists()
```

Comment:
No longer relevant - we don't try to detect the presence of assemblies or additional files any more.